docs/guide/algob-console.md

```diff
@@ -201,11 +201,11 @@ Similar example can be found in `/scrips/transfer/tesla-to-john.js` (tesla ASA).
 ## Transfer Algos according to ASC logic (Contract Account)
 
 Here we will transfer some `algos` from a stateless smart contract ([`/assets/teal/2-gold-contract-asc.teal`](https://github.com/scale-it/algo-builder/blob/develop/examples/asa/assets/teal/2-gold-contract-asc.teal)) to `john`.
-+ We will first load the logic signature (using `deployer.loadLogic(<file_name>.teal)` and get it's address(`lsig.address()`).
++ We will first load the logic signature (using `deployer.loadLogicByFile(<file_name>.teal)` and get it's address(`lsig.address()`).
```

Comment:
Actually we don't have a `loadLogic` now. Because `loadLogic` compiles the contract in real time, and "loads" the logic signature. User should use `getLsig` if he is trying to load logic signature by name. Will add a note here.